llvm/lib/Transforms/Scalar/LowerMatrixIntrinsics.cpp

```diff
@@ -1056,19 +1056,24 @@ class LowerMatrixIntrinsics {
 
       IRBuilder<> Builder(Inst);
 
+      const ShapeInfo &SI = ShapeMap.at(Inst);
+
       if (CallInst *CInst = dyn_cast<CallInst>(Inst))
-        Changed |= VisitCallInst(CInst);
+        Changed |= tryVisitCallInst(CInst);
```

Comment:
I think this should also never fail I think, we should only have shape info for calls we can lower here. If not, `ShapeMap.at()` above would fail.